### ISSUE TITLE
MySensors V_LIGHT_LEVEL should be '%' not 'Lux'

### ISSUE THREAD

**Issue by TheGuv** [domoticz/domoticz#1737]
Dear Devs,

I don't think it matters, but currently running 
```  
  Version: 3.8143
  Build Hash: c57e9862
  Compile Date: 2017-07-27 15:32:14
```

I've just started dev of a MySensors based roller-blind node with integral light sensors which is working fine, but I do notice that the value in Domoticz for the V_LIGHT_LEVEL report has 'Lux' as the unit. This should really be percentage not Lux.

[https://www.mysensors.org/download/serial_api_20](https://www.mysensors.org/download/serial_api_20) shows:
```
V_LIGHT_LEVEL (uncalibrated percentage), V_LEVEL (light level in lux)
```

In summary the unit of V_LIGHT_LEVEL should be "percentage" while V_LEVEL should be "Lux".

Would it be possible to fix this?

Many thanks for all the work.

'Guv

**Comment by gizmocuz** [comment]
Didn't you recently also posted this also on the forum ?

**Comment by gizmocuz** [comment]
Will be changed in the upcoming beta (will take an hour) 8196...
https://github.com/domoticz/domoticz/commit/bef266c0e3cd6c59ae9042661ffaf958167c705e

**Comment by TheGuv** [comment]
Yes, I posted in the forum here http://www.domoticz.com/forum/viewtopic.php?f=6&t=18577 
but wasn't sure if it should also be raised as a Github issue as well.

**Comment by TheGuv** [comment]
Updated to 3.8197 and after removing the existing device (Type: Lux, Lux) and adding the newly created device (Type: General, Percentage), it does indeed display and log as a percentage.Great! 

Many thanks :D 

'guv

**Comment by gizmocuz** [comment]
Thanks for the feedback
